### DISCUSSION TITLE
Scrolling sidebar

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/yari#3330] [Q&A] [answered]
In https://github.com/mdn/yari/issues/2972 @wbamberg brought up an interesting topic. What if our sidebars were sticky as you scroll on desktop. This sounds really interesting.

I played around with the idea in dev tools and recorder a little animated GIF. What do you think? Implementation will not be super hard so, it is more of a UX question.

Please share your thoughts and vote if you would like to see this become a feature of MDN Web Docs.

![An animated GIF showing what a sticky sidebar could look like on MDN Web Docs](https://user-images.githubusercontent.com/10350960/112494122-6d615d00-8d8b-11eb-8eff-893106eccda8.gif)

**Comment by wbamberg** [comment]
I think this is great, thanks for trying it out @schalkneethling ! If you had a branch where we could play around with it I'd love that.

(things I love about this: if I've scrolled some way down the doc and I want to look at compat, I don't have to scroll back up to the top to see the link to BCD).

A lot of sites (e.g. [Stripe](https://stripe.com/docs/api), [GitHub](https://docs.github.com/en/github/authenticating-to-github/signing-commits)) arrange things so the sidebar gets the whole height of the window, and the header is made accordingly narrower. But this is a bigger change and one we may not be ready for.

**Reply by peterbe** [reply]
> If you had a branch where we could play around with it I'd love that.

@schalkneethling If you have a branch, push it to `mdn` (rather than your fork) than we can deploy a whole site with it to the Dev server. It'll get its own domain and everything.

**Comment by schalkneethling** [comment, marked as answer]
This landed as part of the MDN Web Docs redesign 🎉
